x-pack/plugins/infra/public/pages/metrics/hosts/components/tabs/metrics/metrics_grid.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiFlexGrid, EuiFlexItem, EuiFlexGroup, EuiText, EuiI18n } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { MetricChart } from './metric_chart';
+
+const DEFAULT_BREAKDOWN_SIZE = 20;
+
+export const MetricsGrid = () => {
+  return (
+    <>
+      <EuiFlexGroup direction="column" gutterSize="s" data-test-subj="hostsView-metricChart">
+        <EuiFlexItem>
+          <EuiFlexGroup justifyContent="spaceBetween" wrap={false} alignItems="center">
+            <EuiFlexItem grow={false} style={{ flex: 1 }}>
+              <EuiText size="xs">
+                <EuiI18n
+                  token="xpack.infra.hostsViewPage.tabs.metricsCharts.cpuCores"
+                  default="Showing for Top {maxHosts} hosts by {attribute}"
+                  values={{
+                    maxHosts: <strong>{DEFAULT_BREAKDOWN_SIZE}</strong>,
+                    attribute: <strong>name</strong>,
```

Comment:
**question:** should this text be translated too?

Reply:
`name` is the name of the attribute. This can't be translated. In the future, this will react to the table columns sorting.